src/sensors.h

```diff
@@ -5,6 +5,9 @@
 #include <DFRobot_PH.h>
 #include <DHTesp.h>
 #include <OneWire.h>
+#ifdef HAVE_AHT20
+#include <DFRobot_AHT20.h>
+#endif
 
```

Comment:
Two be able to do this, we need to move `#include "config.h"` to the first line of the file. Otherwise, `#ifdef HAVE_AHT20` will always be false. I think I missed doing the same for the other includes.